src/main/java/net/sf/rails/ui/swing/StartRoundWindow.java

```diff
@@ -361,56 +345,45 @@ private void initCells() {
                 playerFreeCashXOffset - 1, playerFreeCashYOffset, 1, 1,
                 WIDE_RIGHT + (firstBelowTable ? WIDE_TOP : 0));
         for (int i = 0; i < np; i++) {
-            f =
-                    playerFree[i] =
-                            new Field(includeBidding
-                                    ? round.getFreeCashModel(players.getPlayerByPosition(i))
-                                    : players.getPlayerByPosition(i).getWallet());
-            addField(f, playerFreeCashXOffset + i, playerFreeCashYOffset, 1, 1,
-                    firstBelowTable ? WIDE_TOP : 0);
+            playerFree[i] = new Field(includeBidding
+                    ? round.getFreeCashModel(players.getPlayerByPosition(i))
+                    : players.getPlayerByPosition(i).getWallet());
+            addField(playerFree[i], playerFreeCashXOffset + i, playerFreeCashYOffset, 1, 1, firstBelowTable ? WIDE_TOP : 0);
         }
 
         for (int i = 0; i < np; i++) {
-            f = lowerPlayerCaption[i] = new Field(players.getPlayerByPosition(i).getPlayerNameModel());
-            addField(f, playerFreeCashXOffset + i, playerFreeCashYOffset + 1,
-                    1, 1, WIDE_TOP);
+            lowerPlayerCaption[i] = new Field(players.getPlayerByPosition(i).getPlayerNameModel());
+            addField(lowerPlayerCaption[i], playerFreeCashXOffset + i, playerFreeCashYOffset + 1, 1, 1, WIDE_TOP);
         }
 
         dummyButton = new ClickField("", "", "", this, itemGroup);
```

Comment:
Similar to the field `itemStatus` this field does not seem to be added to the UI instead it is only added to a button group and preselected at the beginning of `updateStatus`. What is this button used for, do we require it?

Reply:
It is being added to the UI-- it's added via the `addField()` method on line 356 and it's manipulated during game play (highlighting) like this:

<img width="285" alt="Screen Shot 2020-05-18 at 6 13 06 PM" src="https://user-images.githubusercontent.com/6716825/82267938-60860880-9933-11ea-9a4a-b36e07c5ce5b.png">

Reply:
Interesting Picture there is already missing a field ? whats the item 1?

Reply:
Furthermore by removing the actions you are removing the replayability i think.

Reply:
@neutronc that is 18EU... the minors are numbered, not named. The picture is correct ;)

Reply:
You misunderstood my comment.

I didn't mean `lowerPlayerCaption`, I meant the `ClickField` object `dummyButton`.
According to my IntelliJ this field is only accessed twice in the Code:
- in line 359 where it is defined
- in line 388 where it is set as selected

Except these two occurrences the field is never used.

Reply:
Seems it's purpose is clarified by the comment on line 387 and comment on line 114?

Now whether it's actually need to accomplish what those two comments indicate is the question-- which you'd have to manually test I think to verify.

Reply:
Yes I guess I don't get around testing this.

For now I think it is safe to ignore the `dummyButton` field and let it remain part of the class.
I think I will remove the field when we change the window to a JavaFX implementation.

Reply:
Pleas remember that you also need to change and test all inherited Classes with Game Specific implementations.
And dont assume that there are no changes in certain fields that are static in the base game (like the player action order in the startroundWindow) Subsequent StartRounds might have a different PlayerOrder (1880 Investor is an example).

Reply:
> Pleas remember that you also need to change and test all inherited Classes with Game Specific implementations.

I need your help testing all the game specific implementations, because I don't know how they should behave. I need people that know how these game modes should work to evaluate whether this PR contains any regressions in this regard. Therefore it is also fine if this PR takes a while until it is merged.

> And dont assume that there are no changes in certain fields that are static in the base game (like the player action order in the startroundWindow) Subsequent StartRounds might have a different PlayerOrder (1880 Investor is an example).

I normally don't assume stuff. I base my changes on observations in the code. If I'm not sure whether a change is correct I ask other developers about their opinion (for example see https://github.com/Rails-18xx/Rails/pull/216#discussion_r425347239). 

If you see additional changes you think are dangerous in anyway and should be further analyzed please comment on them. This comment is about an unchanged line which because it is unchanged does not lead to a regression.